kanopi/patternlab-hjf
# 26830287 fix pa11y errors

## Description
Fix Pa11y errors.

https://circle-production-customer-artifacts.s3.amazonaws.com/picard/605dfef12d56914b3a169f72/60dc941d1a53571ab1f631bf-0-build/artifacts/tmp/pa11y-ci-results.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210630T160820Z&X-Amz-SignedHeaders=host&X-Amz-Expires=59&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210630%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=0644c9024b8c913e8d640cc85883bf8e2946ec79215948b9098b6c57a9c400ee

Goes with this HJFMRI pull request: https://github.com/kanopi/hjf_mri/pull/8

## Affected URL
[https://pr-8-hjfmri.pantheonsite.io/](https://pr-8-hjfmri.pantheonsite.io/)


## Related Tickets

* [Correct PA11Y Errors](https://kanopi.teamwork.com/#/tasks/26830287)


## Steps to Validate
1. After pull request for site is created, CircleCI will run pa11y audit.
2. Confirm fix in pa11y audit result file.

## Deploy Notes
